refactor(accounts-management)!: enhance account management functionality

saledjenic opened the pull request:
The main goal of this change is to internally (for the account manager) change the keystore based on the selected chat account. That simplifies the usage of the account manager across the app. Main changes are made in `AccountsManager` and all other changes reflect changes made there.

- Updated `AccountsManager` to support setting a root data directory.
- Refactored methods to let the client deal with the keystore transparently.
- Adjusted tests to reflect changes in account creation and password verification processes.
- Removed unnecessary keystore directory references from various components.
- Unused functions/endpoints removed:
  - `SaveAccountAndStartNodeWithKey`
  - `InitKeystore`
  - `DeleteMultiaccount`
  - `DeleteMultiaccountV2`
  - `SaveAccountAndLoginWithKeycard`

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (150)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/1/) | 2025-07-03 17:09:10 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250703-170645-29c95f-pr6732.aar) |
| :heavy_multiplication_x: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/1/) | 2025-07-03 17:09:44 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/1//consoleText) |
| :heavy_check_mark: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/1/) | 2025-07-03 17:09:49 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250703-170653-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/1/) | 2025-07-03 17:09:59 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250703-170648-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/1/) | 2025-07-03 17:10:52 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250703-170656-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/1/) | 2025-07-03 17:11:55 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250703-170646-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/1/) | 2025-07-03 17:12:47 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250703-170700-29c95f-pr6732.zip) |
| :heavy_multiplication_x: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/1/) | 2025-07-03 17:13:18 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/1//consoleText) |
| :heavy_check_mark: | 29c95fd7 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/1/) | 2025-07-03 17:17:07 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250703-170650-29c95f-pr6732.zip) |
| :heavy_multiplication_x: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/2/) | 2025-07-04 06:50:29 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/2//consoleText) |
| :heavy_check_mark: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/2/) | 2025-07-04 06:50:45 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250704-064757-29c95f-pr6732.aar) |
| :heavy_check_mark: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/2/) | 2025-07-04 06:50:45 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-064802-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/2/) | 2025-07-04 06:51:13 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-064759-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/2/) | 2025-07-04 06:51:54 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-064804-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/2/) | 2025-07-04 06:52:09 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250704-064758-29c95f-pr6732.zip) |
| :heavy_check_mark: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/2/) | 2025-07-04 06:53:45 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250704-064809-29c95f-pr6732.zip) |
| :heavy_multiplication_x: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/2/) | 2025-07-04 06:54:07 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/2//consoleText) |
| :heavy_check_mark: | 29c95fd7 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/2/) | 2025-07-04 06:57:14 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-064800-29c95f-pr6732.zip) |
| | | | | | | |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/3/) | 2025-07-04 10:12:20 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250704-101010-7c1548-pr6732.aar) |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/3/) | 2025-07-04 10:13:06 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-101012-7c1548-pr6732.zip) |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/3/) | 2025-07-04 10:13:15 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-101017-7c1548-pr6732.zip) |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/3/) | 2025-07-04 10:14:11 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-101020-7c1548-pr6732.zip) |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/3/) | 2025-07-04 10:14:30 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250704-101011-7c1548-pr6732.zip) |
| :heavy_multiplication_x: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/3/) | 2025-07-04 10:15:55 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/3//consoleText) |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/3/) | 2025-07-04 10:16:06 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250704-101024-7c1548-pr6732.zip) |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/3/) | 2025-07-04 10:20:55 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-101014-7c1548-pr6732.zip) |
| :heavy_check_mark: | 7c154867 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/3/) | 2025-07-04 10:35:06 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0814426d | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/4/) | 2025-07-04 10:17:48 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250704-101452-081442-pr6732.aar) |
| :heavy_check_mark: | 0814426d | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/4/) | 2025-07-04 10:17:49 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-101459-081442-pr6732.zip) |
| :heavy_check_mark: | 0814426d | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/4/) | 2025-07-04 10:17:58 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-101455-081442-pr6732.zip) |
| :heavy_check_mark: | 0814426d | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/4/) | 2025-07-04 10:19:02 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-101502-081442-pr6732.zip) |
| :heavy_check_mark: | 0814426d | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/4/) | 2025-07-04 10:19:11 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250704-101454-081442-pr6732.zip) |
| :heavy_multiplication_x: | 0814426d | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/4/) | 2025-07-04 10:21:24 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/4//consoleText) |
| :heavy_check_mark: | 0814426d | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/4/) | 2025-07-04 10:21:41 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250704-101609-081442-pr6732.zip) |
| | | | | | | |
| :heavy_check_mark: | f4d97904 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/5/) | 2025-07-04 10:20:40 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-101754-f4d979-pr6732.zip) |
| :heavy_check_mark: | f4d97904 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/5/) | 2025-07-04 10:20:47 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250704-101756-f4d979-pr6732.aar) |
| :heavy_check_mark: | f4d97904 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/5/) | 2025-07-04 10:20:57 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-101802-f4d979-pr6732.zip) |
| :heavy_check_mark: | f4d97904 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/5/) | 2025-07-04 10:23:02 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-101910-f4d979-pr6732.zip) |
| :heavy_check_mark: | f4d97904 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/5/) | 2025-07-04 10:23:31 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250704-101919-f4d979-pr6732.zip) |
| | | | | | | |
| :heavy_check_mark: | 8de7548f | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/6/) | 2025-07-04 10:23:14 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250704-102057-8de754-pr6732.aar) |
| :heavy_check_mark: | 8de7548f | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/6/) | 2025-07-04 10:23:31 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-102045-8de754-pr6732.zip) |
| :heavy_check_mark: | 8de7548f | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/6/) | 2025-07-04 10:23:50 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-102104-8de754-pr6732.zip) |
| :heavy_multiplication_x: | 8de7548f | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/5/) | 2025-07-04 10:26:43 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/5//consoleText) |
| :heavy_check_mark: | 8de7548f | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/6/) | 2025-07-04 10:27:01 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250704-102312-8de754-pr6732.zip) |
| :heavy_check_mark: | 8de7548f | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/5/) | 2025-07-04 10:27:07 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250704-102144-8de754-pr6732.zip) |
| :heavy_check_mark: | 8de7548f | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/6/) | 2025-07-04 10:27:58 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250704-102344-8de754-pr6732.zip) |
| :heavy_check_mark: | 8de7548f | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/4/) | 2025-07-04 10:28:53 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250704-102059-8de754-pr6732.zip) |
| :heavy_check_mark: | 8de7548f | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/4/) | 2025-07-04 11:00:49 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 681fc56e | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/7/) | 2025-07-21 15:56:04 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250721-155338-681fc5-pr6732.aar) |
| :heavy_check_mark: | 681fc56e | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/7/) | 2025-07-21 15:56:34 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-155344-681fc5-pr6732.zip) |
| :heavy_check_mark: | 681fc56e | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/7/) | 2025-07-21 15:56:51 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-155341-681fc5-pr6732.zip) |
| :heavy_check_mark: | 681fc56e | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/7/) | 2025-07-21 15:58:08 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250721-155339-681fc5-pr6732.zip) |
| :heavy_check_mark: | 681fc56e | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/7/) | 2025-07-21 15:58:52 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-155347-681fc5-pr6732.zip) |
| :heavy_check_mark: | 681fc56e | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/6/) | 2025-07-21 15:59:33 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250721-155352-681fc5-pr6732.zip) |
| :heavy_multiplication_x: | 681fc56e | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/6/) | 2025-07-21 16:00:13 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/6//consoleText) |
| :heavy_check_mark: | 681fc56e | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/5/) | 2025-07-21 16:03:43 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-155342-681fc5-pr6732.zip) |
| :heavy_check_mark: | 681fc56e | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/5/) | 2025-07-21 16:18:00 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 3afc8ed3 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/8/) | 2025-07-23 08:01:56 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-075902-3afc8e-pr6732.zip) |
| :heavy_check_mark: | 3afc8ed3 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/8/) | 2025-07-23 08:01:56 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250723-075854-3afc8e-pr6732.aar) |
| :heavy_check_mark: | 3afc8ed3 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/8/) | 2025-07-23 08:02:03 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-075857-3afc8e-pr6732.zip) |
| :heavy_multiplication_x: | 3afc8ed3 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/7/) | 2025-07-23 08:02:33 | ~3 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/7//consoleText) |
| :heavy_check_mark: | 3afc8ed3 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/8/) | 2025-07-23 08:03:29 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250723-075855-3afc8e-pr6732.zip) |
| :heavy_check_mark: | 3afc8ed3 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/8/) | 2025-07-23 08:04:06 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-075904-3afc8e-pr6732.zip) |
| :heavy_check_mark: | 3afc8ed3 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/7/) | 2025-07-23 08:05:04 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250723-075908-3afc8e-pr6732.zip) |
| :heavy_check_mark: | 3afc8ed3 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/6/) | 2025-07-23 08:09:29 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-075858-3afc8e-pr6732.zip) |
| :heavy_check_mark: | 3afc8ed3 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/6/) | 2025-07-23 08:23:37 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 6a648a63 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/9/) | 2025-07-23 08:04:12 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250723-080203-6a648a-pr6732.aar) |
| :heavy_check_mark: | 6a648a63 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/9/) | 2025-07-23 08:04:42 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-080201-6a648a-pr6732.zip) |
| :heavy_check_mark: | 6a648a63 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/9/) | 2025-07-23 08:05:04 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-080209-6a648a-pr6732.zip) |
| :heavy_check_mark: | 6a648a63 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/9/) | 2025-07-23 08:09:01 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-080411-6a648a-pr6732.zip) |
| :heavy_check_mark: | 6a648a63 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/9/) | 2025-07-23 08:09:58 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250723-080338-6a648a-pr6732.zip) |
| :heavy_multiplication_x: | 6a648a63 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/8/) | 2025-07-23 08:10:07 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/8//consoleText) |
| :heavy_check_mark: | 6a648a63 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/8/) | 2025-07-23 08:10:48 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250723-080507-6a648a-pr6732.zip) |
| :heavy_check_mark: | 6a648a63 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/7/) | 2025-07-23 08:11:16 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-080040-6a648a-pr6732.zip) |
| :heavy_check_mark: | 6a648a63 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/7/) | 2025-07-23 08:48:49 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 98b2dedd | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/10/) | 2025-07-23 15:44:01 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250723-154129-98b2de-pr6732.aar) |
| :heavy_check_mark: | 98b2dedd | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/10/) | 2025-07-23 15:44:18 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-154136-98b2de-pr6732.zip) |
| :heavy_check_mark: | 98b2dedd | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/10/) | 2025-07-23 15:44:38 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-154132-98b2de-pr6732.zip) |
| :heavy_multiplication_x: | 98b2dedd | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/8/) | 2025-07-23 15:45:29 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/8//consoleText) |
| :heavy_check_mark: | 98b2dedd | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/10/) | 2025-07-23 15:45:43 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250723-154130-98b2de-pr6732.zip) |
| :heavy_check_mark: | 98b2dedd | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/10/) | 2025-07-23 15:46:38 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-154139-98b2de-pr6732.zip) |
| :heavy_check_mark: | 98b2dedd | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/9/) | 2025-07-23 15:47:26 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250723-154143-98b2de-pr6732.zip) |
| :heavy_multiplication_x: | 98b2dedd | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/9/) | 2025-07-23 15:47:40 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/9//consoleText) |
| :heavy_check_mark: | 98b2dedd | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/8/) | 2025-07-23 15:51:00 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-154133-98b2de-pr6732.zip) |
| | | | | | | |
| :heavy_check_mark: | eb5bb21e | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/11/) | 2025-07-24 06:40:51 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-063754-eb5bb2-pr6732.zip) |
| :heavy_check_mark: | eb5bb21e | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/11/) | 2025-07-24 06:40:58 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250724-063747-eb5bb2-pr6732.aar) |
| :heavy_check_mark: | eb5bb21e | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/11/) | 2025-07-24 06:41:00 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-063750-eb5bb2-pr6732.zip) |
| :heavy_check_mark: | eb5bb21e | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/11/) | 2025-07-24 06:42:05 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250724-063749-eb5bb2-pr6732.zip) |
| :heavy_check_mark: | eb5bb21e | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/11/) | 2025-07-24 06:43:03 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-063758-eb5bb2-pr6732.zip) |
| :heavy_multiplication_x: | eb5bb21e | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/10/) | 2025-07-24 06:43:50 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/10//consoleText) |
| :heavy_check_mark: | eb5bb21e | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/10/) | 2025-07-24 06:44:37 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250724-063802-eb5bb2-pr6732.zip) |
| :heavy_check_mark: | eb5bb21e | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/9/) | 2025-07-24 06:47:41 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-063751-eb5bb2-pr6732.zip) |
| :heavy_check_mark: | eb5bb21e | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/9/) | 2025-07-24 07:02:09 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 5705d26f | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/12/) | 2025-07-24 07:53:58 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250724-075056-5705d2-pr6732.aar) |
| :heavy_check_mark: | 5705d26f | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/12/) | 2025-07-24 07:54:03 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-075103-5705d2-pr6732.zip) |
| :heavy_check_mark: | 5705d26f | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/12/) | 2025-07-24 07:54:07 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-075059-5705d2-pr6732.zip) |
| :heavy_multiplication_x: | 5705d26f | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/10/) | 2025-07-24 07:55:06 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/10//consoleText) |
| :heavy_check_mark: | 5705d26f | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/12/) | 2025-07-24 07:55:09 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250724-075058-5705d2-pr6732.zip) |
| :heavy_check_mark: | 5705d26f | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/12/) | 2025-07-24 07:56:17 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-075107-5705d2-pr6732.zip) |
| :heavy_check_mark: | 5705d26f | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/11/) | 2025-07-24 07:56:51 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250724-075111-5705d2-pr6732.zip) |
| :heavy_check_mark: | 5705d26f | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/11/) | 2025-07-24 07:57:54 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/11//consoleText) |
| :heavy_check_mark: | 5705d26f | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/10/) | 2025-07-24 08:00:30 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-075101-5705d2-pr6732.zip) |
| :heavy_multiplication_x: | 5705d26f | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/11/) | 2025-07-24 08:01:12 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/11//consoleText) |
| :heavy_multiplication_x: | 5705d26f | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/12/) | 2025-07-24 08:09:48 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e7294cf2 | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/13/) | 2025-07-24 08:13:18 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250724-081030-e7294c-pr6732.aar) |
| :heavy_check_mark: | e7294cf2 | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/13/) | 2025-07-24 08:13:28 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-081033-e7294c-pr6732.zip) |
| :heavy_multiplication_x: | e7294cf2 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/13/) | 2025-07-24 08:14:51 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/13//consoleText) |
| :heavy_check_mark: | e7294cf2 | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/13/) | 2025-07-24 08:15:41 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-081040-e7294c-pr6732.zip) |
| :heavy_check_mark: | e7294cf2 | [#12](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/12/) | 2025-07-24 08:16:30 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250724-081045-e7294c-pr6732.zip) |
| :heavy_check_mark: | e7294cf2 | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/14/) | 2025-07-24 08:16:38 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-081352-e7294c-pr6732.zip) |
| :heavy_check_mark: | e7294cf2 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/12/) | 2025-07-24 08:17:16 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/12//consoleText) |
| :heavy_check_mark: | e7294cf2 | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/14/) | 2025-07-24 08:17:56 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250724-081345-e7294c-pr6732.zip) |
| :heavy_check_mark: | e7294cf2 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/11/) | 2025-07-24 08:18:09 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-081035-e7294c-pr6732.zip) |
| | | | | | | |
| :heavy_check_mark: | dd0cb9b4 | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/14/) | 2025-07-24 08:41:52 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250724-083933-dd0cb9-pr6732.aar) |
| :heavy_check_mark: | dd0cb9b4 | [#15](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/15/) | 2025-07-24 08:42:25 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-083939-dd0cb9-pr6732.zip) |
| :heavy_check_mark: | dd0cb9b4 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/14/) | 2025-07-24 08:42:47 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-083935-dd0cb9-pr6732.zip) |
| :heavy_check_mark: | dd0cb9b4 | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/15/) | 2025-07-24 08:43:40 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250724-083934-dd0cb9-pr6732.zip) |
| :heavy_check_mark: | dd0cb9b4 | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/14/) | 2025-07-24 08:44:36 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-083942-dd0cb9-pr6732.zip) |
| :heavy_check_mark: | dd0cb9b4 | [#13](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/13/) | 2025-07-24 08:45:27 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250724-083946-dd0cb9-pr6732.zip) |
| :heavy_check_mark: | dd0cb9b4 | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/13/) | 2025-07-24 08:45:47 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/13//consoleText) |
| :heavy_check_mark: | dd0cb9b4 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/12/) | 2025-07-24 08:47:36 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-083937-dd0cb9-pr6732.zip) |
| :heavy_check_mark: | dd0cb9b4 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/14/) | 2025-07-24 09:04:12 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/14//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8831c86e | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/15/) | 2025-07-30 11:11:35 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250730-110906-8831c8-pr6732.aar) |
| :heavy_check_mark: | 8831c86e | [#16](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/16/) | 2025-07-30 11:12:08 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250730-110913-8831c8-pr6732.zip) |
| :heavy_check_mark: | 8831c86e | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/15/) | 2025-07-30 11:12:19 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250730-110908-8831c8-pr6732.zip) |
| :heavy_check_mark: | 8831c86e | [#16](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/16/) | 2025-07-30 11:13:45 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250730-110908-8831c8-pr6732.zip) |
| :heavy_check_mark: | 8831c86e | [#15](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/15/) | 2025-07-30 11:14:25 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250730-110916-8831c8-pr6732.zip) |
| :heavy_check_mark: | 8831c86e | [#14](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/14/) | 2025-07-30 11:15:04 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250730-110921-8831c8-pr6732.zip) |
| :heavy_check_mark: | 8831c86e | [#14](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/14/) | 2025-07-30 11:16:09 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/14//consoleText) |
| :heavy_check_mark: | 8831c86e | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/13/) | 2025-07-30 11:19:47 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250730-110910-8831c8-pr6732.zip) |
| :heavy_multiplication_x: | 8831c86e | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/15/) | 2025-07-30 11:32:59 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/15//consoleText) |
| | | | | | | |
| :heavy_check_mark: | ca10db0e | [#16](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/16/) | 2025-07-30 11:14:04 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250730-111142-ca10db-pr6732.aar) |
| :heavy_check_mark: | ca10db0e | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/16/) | 2025-07-30 11:15:29 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250730-111225-ca10db-pr6732.zip) |
| :heavy_check_mark: | ca10db0e | [#17](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/17/) | 2025-07-30 11:15:54 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250730-111217-ca10db-pr6732.zip) |
| :heavy_check_mark: | ca10db0e | [#17](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/17/) | 2025-07-30 11:18:23 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250730-111401-ca10db-pr6732.zip) |
| :heavy_check_mark: | ca10db0e | [#16](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/16/) | 2025-07-30 11:19:18 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250730-111430-ca10db-pr6732.zip) |
| :heavy_check_mark: | ca10db0e | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/14/) | 2025-07-30 11:19:46 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250730-110957-ca10db-pr6732.zip) |
| :heavy_check_mark: | ca10db0e | [#15](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/15/) | 2025-07-30 11:20:47 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250730-111511-ca10db-pr6732.zip) |
| :heavy_check_mark: | ca10db0e | [#15](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/15/) | 2025-07-30 11:23:36 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/15//consoleText) |
| :heavy_check_mark: | ca10db0e | [#16](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/16/) | 2025-07-30 11:58:31 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/16//consoleText) |
| | | | | | | |
| :heavy_check_mark: | bada58be | [#18](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/18/) | 2025-07-30 17:05:39 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250730-170255-bada58-pr6732.aar) |
| :heavy_check_mark: | bada58be | [#18](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/18/) | 2025-07-30 17:06:01 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250730-170131-bada58-pr6732.zip) |
| :heavy_multiplication_x: | bada58be | [#17](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/17/) | 2025-07-30 17:06:19 | ~3 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/17//consoleText) |
| :heavy_multiplication_x: | bada58be | [#18](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/18/) | 2025-07-30 17:06:39 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/18//consoleText) |
| :heavy_check_mark: | bada58be | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/18/) | 2025-07-30 17:07:10 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250730-170303-bada58-pr6732.zip) |
| :heavy_check_mark: | bada58be | [#17](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/17/) | 2025-07-30 17:07:12 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250730-170126-bada58-pr6732.zip) |
| :heavy_check_mark: | bada58be | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/16/) | 2025-07-30 17:12:34 | ~11 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250730-170116-bada58-pr6732.zip) |
| :heavy_check_mark: | bada58be | [#19](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/19/) | 2025-07-30 17:14:21 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250730-171005-bada58-pr6732.zip) |
| :heavy_check_mark: | bada58be | [#19](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/19/) | 2025-07-30 17:17:25 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250730-171016-bada58-pr6732.zip) |
| :heavy_multiplication_x: | bada58be | [#19](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/19/) | 2025-07-31 06:36:57 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/19//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_multiplication_x: | ccf70c5d | [#20](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/20/) | 2025-07-31 10:58:59 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/20//consoleText) |
| :heavy_check_mark: | ccf70c5d | [#19](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/19/) | 2025-07-31 10:59:49 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250731-105640-ccf70c-pr6732.zip) |
| :heavy_check_mark: | ccf70c5d | [#19](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/19/) | 2025-07-31 11:00:00 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250731-105637-ccf70c-pr6732.aar) |
| :heavy_check_mark: | ccf70c5d | [#19](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/19/) | 2025-07-31 11:01:52 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250731-105647-ccf70c-pr6732.zip) |
| :heavy_check_mark: | ccf70c5d | [#18](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/18/) | 2025-07-31 11:02:34 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250731-105652-ccf70c-pr6732.zip) |
| :heavy_multiplication_x: | ccf70c5d | [#18](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/18/) | 2025-07-31 11:03:50 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/18//consoleText) |
| :heavy_check_mark: | ccf70c5d | [#17](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/17/) | 2025-07-31 11:06:32 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250731-105642-ccf70c-pr6732.zip) |
| :heavy_check_mark: | ccf70c5d | [#20](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/20/) | 2025-07-31 11:09:45 | ~13 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250731-105639-ccf70c-pr6732.zip) |
| :heavy_check_mark: | ccf70c5d | [#20](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/20/) | 2025-07-31 11:14:23 | ~17 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250731-105644-ccf70c-pr6732.zip) |
| | | | | | | |
| :heavy_check_mark: | 7762cd87 | [#20](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6732/20/) | 2025-07-31 11:33:24 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250731-112950-7762cd-pr6732.aar) |
| :heavy_check_mark: | 7762cd87 | [#20](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6732/20/) | 2025-07-31 11:33:43 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250731-112952-7762cd-pr6732.zip) |
| :heavy_check_mark: | 7762cd87 | [#19](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6732/19/) | 2025-07-31 11:35:43 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250731-113004-7762cd-pr6732.zip) |
| :heavy_check_mark: | 7762cd87 | [#20](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6732/20/) | 2025-07-31 11:37:50 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250731-113001-7762cd-pr6732.zip) |
| :heavy_multiplication_x: | 7762cd87 | [#21](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/21/) | 2025-07-31 11:39:45 | ~9 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/21//consoleText) |
| :heavy_multiplication_x: | 7762cd87 | [#19](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/19/) | 2025-07-31 11:40:07 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6732/19//consoleText) |
| :heavy_check_mark: | 7762cd87 | [#18](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6732/18/) | 2025-07-31 11:42:55 | ~13 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250731-112954-7762cd-pr6732.zip) |
| :heavy_check_mark: | 7762cd87 | [#22](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/22/) | 2025-07-31 12:06:02 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6732/22//consoleText) |
| :heavy_check_mark: | 7762cd87 | [#21](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6732/21/) | 2025-07-31 12:07:59 | ~38 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250731-112952-7762cd-pr6732.zip) |
| :heavy_check_mark: | 7762cd87 | [#21](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6732/21/) | 2025-07-31 12:09:29 | ~39 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250731-112957-7762cd-pr6732.zip) |

saledjenic committed:
refactor(accounts-management)!: enhance account management functionality

The main goal of this change is to internally (for the account manager) change the keystore based on the selected
chat account. That simplifies a usage of the account manager across the app. Main changes are made in
`AccountsManager` and all other changes reflect changes made there.

- Updated `AccountsManager` to support setting a root data directory.
- Refactored methods to let the client deals with the keystore transparently.
- Adjusted tests to reflect changes in account creation and password verification processes.
- Removed unnecessary keystore directory references from various components.
- Unused functions/endpoints removed:
- `SaveAccountAndStartNodeWithKey`
- `InitKeystore`
- `DeleteMultiaccount`
- `DeleteMultiaccountV2`
- `SaveAccountAndLoginWithKeycard`

saledjenic commented:
I've sorted out functional and unit tests affected by the changes done in `accounts-management` part of the app. Also tested the desktop app and all looks good. Please have a look/review the code and if you have time, feel free to test the app.

osmaczko reviewed (dismissed):
Huge work 🚀 

Commit should be marked with `!` - client facing APIs has changed.

igor-sirotin commented on accounts-management/accounts.go:
```diff
@@ -168,43 +212,69 @@ func (m *AccountsManager) storeToKeystore(acc *generator.Account, password strin
-func (m *AccountsManager) SetChatAccountWithPrivateKey(privKey *ecdsa.PrivateKey) error {
-	account := generator.NewAccount(privKey, nil)
+// SetChatAccount sets the chat account and keystore either by address and password or by private key
+func (m *AccountsManager) SetChatAccount(address ethtypes.Address, password string, privateKey *ecdsa.PrivateKey) error {
```
Should the mutex not be locked directly here?
You're locking it in `isChatAccountSet`, `setKeystore` and `setChatAccount`, but what if the `SetChatAccount` is called concurrently? (not sure if it's a realistic example, but you're adding mutex for some reason, so I assume it's possible)

Didn't check, but probably applies to all public methods here.

saledjenic replied:
The function itself doesn't access shared resources, instead, it calls `isChatAccountSet`, `createKeystore`, `setKeystore` and `setChatAccount` and each of those private functions has locking.

igor-sirotin replied:
Sure. But if `SetChatAccount` is called concurrently at the same time, then the behaviour may not by consistent. 

E.g. if you call `SetChatAccount` with the same new address, then both calls may pass `isChatAccountSet` just fine and save the same address twice.

saledjenic replied:
Yes, I see what you mean. Need to think, check that.

saledjenic replied:
@igor-sirotin I went through the code and moved some locks so the accounts manager should be thread-safe.

igor-sirotin commented on accounts-management/accounts.go:
```diff
@@ -66,18 +101,27 @@ func (m *AccountsManager) CreateAndStoreAccount(password string) (genAccount *ge
 // CreateFromMnemonicAndStoreAccount creates an internal geth account from a mnemonic and stores it in the keystore
-func (m *AccountsManager) CreateFromMnemonicAndStoreAccount(mnemonic string, password string) (genAccount *generator.Account, err error) {
+// if profile is true means that a profile keypair is being created, and the keystore is set to the new one
+func (m *AccountsManager) CreateFromMnemonicAndStoreAccount(mnemonic string, password string, profile bool) (genAccount *generator.Account, err error) {
```
It seems that `profile` argument is `false` in most cases. I think it would be better to explicitly call `ReloadKeystore` instead of setting `profile=true`. 

(sorry classic boolean args bike shedding)

saledjenic replied:
This is not the final change. You can already see in this https://github.com/status-im/status-go/pull/6777, draft PR, which is on top of this one, a final usage of it (though it's modified a bit and renamed it to `CreateKeypairFromMnemonicAndStore`).

igor-sirotin commented on accounts-management/accounts_test.go:
```diff
@@ -23,81 +25,102 @@ import (
+		if testCase.keystoreSet {
+			accManager.setKeystore(keystore)
+
+			time.Sleep(keystoreReloadDelay)
```
😮 

Any chance we can avoid sleeping in tests?

saledjenic replied:
That doesn't happen in the app, but in tests we're verifying password and the thing is that we set a keystore and in the next like the test is verifying the password by checking to decrypt a local keystore file. The thing is that on the go-ethereum side, once the keystore is set, accounts are loaded in an async way to the local cache and that's why I added that sleep to simulate real uasge (give some time for the keystore cache to load).

igor-sirotin replied:
I understand what you did, but relying on `time.Sleep` in tests is the #1 case to make them flaky.
If for whatever reason (slower/busy host) this procedure takes more time, the test will fail.

Is there a way to deterministically know that the accounts are loaded?

saledjenic replied:
It was something else, not when creating a keystore, maybe I just forgot to remove sleep. It should be fine even without it. Will commit the change now.

saledjenic replied:
@igor-sirotin that change I did a few weeks ago and forgot what the real reason why I added the sleep, but I checked it now and recall what it was, so going to explain now.

It's all about moving the keystore files directly to the keystore location, by copying the file in order to do the test with a certain keystore file.

When moving keystore files, by copying directly (not via keystore instance), to the keystore dir which is registered to the go-ethereum keystore, a consumer (in this case our test) needs to wait until the "reload" algorithm of the go-ethereum keystore re-scan the keystore dir and register all keystore files to its cache. That was solved by the `time.Sleep` call.

In the second commit here in this PR I removed `time.Sleep` and reloaded (re-created) go-ethereum keystore instance after keystore file is copied to the keystore dir. This way a newly create keystore instance is aware of keystore file that was manually copied.

igor-sirotin commented on accounts-management/README.md:
```diff
@@ -169,12 +176,21 @@ func main() {
+## Persistence Interface Requirements
+
+Your `Persistence` implementation must provide:
+- `AddressExists(address ethtypes.Address) (bool, error)`
+- `GetProfileKeypair() (*accounts.Keypair, error)`
+- `GetWalletRootAddress() (ethtypes.Address, error)`
+- `GetPath(address ethtypes.Address) (string, error)`
```
Why are there no setters? 🤔

igor-sirotin replied:
Also, this doesn't seem to match the code (minor comment ofc)
https://github.com/status-im/status-go/blob/1302a6a9c414ec635026e4e0e8a9fd08449156a6/accounts-management/persistence.go#L9-L16

saledjenic replied:
That's it for this stage, you can see in this https://github.com/status-im/status-go/pull/6777, draft PR, that I've already added more functions there.

igor-sirotin commented on api/geth_backend.go:
```diff
@@ -1772,49 +1648,52 @@ func (b *GethStatusBackend) prepareSettings(request *requests.CreateAccount, inp
+		KeyUID:             keypair.KeyUID,
 		Address:            types.HexToAddress(walletDerivedAccount.Address),
 		ColorID:            multiacccommon.CustomizationColor(request.CustomizationColor),
-		Emoji:              emoji,
```
What happened with `emoji`, we don't use it anymore?

saledjenic replied:
We use it, but at this step, we're setting up the default wallet account. It's the Status user creation step and the default wallet account is always a wallet icon for it.

igor-sirotin commented on api/test_helpers.go:
```diff
@@ -2,98 +2,167 @@ package api
+type setupContext struct {
+	backend        *GethStatusBackend
+	mnemonic       string
+	settings       settings.Settings
```
```suggestion
	settings       *settings.Settings
```

saledjenic replied:
Will be changed in the follow-up PR, here https://github.com/status-im/status-go/pull/6777

igor-sirotin commented on nodecfg/node_config.go:
```diff
@@ -418,6 +418,7 @@ func migrateNodeConfig(tx *sql.Tx) error {
 func loadNodeConfig(tx *sql.Tx) (*params.NodeConfig, error) {
 	nodecfg := &params.NodeConfig{}
 
+	keystoreDir := "" // TODO: remove this from db
```
Can't we just remove it from the SQL select query?

saledjenic replied:
Ofc we can, but will still live in the table. This way reminds us to fully clean it, once.

saledjenic commented:
@osmaczko @igor-sirotin thanks for the review.

saledjenic committed:
test_: removed sleep

When moving keystore files to the keystore dir which is registered to the go-ethereum keystore, the app needs to
wait until the "reload" algorithm of the go-ethereum keystore re-scan the keystore dir and register all keystore files
to its cache. That was solved by the `time.Sleep` call.

This PR removes `time.Sleep` and instead reload (re-create) go-ethereum keystore after keystore files are copied
to the keystore dir.
